man/lo2s.1.pod

```diff
@@ -154,6 +154,14 @@ determine the trace path instead, including variable substitution.
 Attach to a running process with process ID I<PID> instead of launching
 I<COMMAND>.
 
+=item B<-u>, B<--as-current-user>
+
+Launch I<COMMAND> as the current user. Requires lo2s call with sudo.
+
+=item B<-U>, B<--as-user> I<USERNAME>
+
+Launch I<COMMAND> as the specified user. Implies B<-u> but does not require sudo, but the current user has to have setuid() privileges. Will take priority over B<-u> if specified together.
```

Comment:
user I<USERNAME> instead of "the specified user" . I think the part about "implies" in its current implementation confuses more than it helps